### DISCUSSION TITLE
Ephemeral labels not being respected?

### DISCUSSION THREAD

**Discussion by stalinCCCP415** [almeidapaulopt/tsdproxy#71] [Q&A] [answered]
I've been playing around with v1.0.0. In previous releases I generated an auth key with ephemeral enabled, and all proxies were set up as ephemeral nodes. 

For playing around with 1.0.0, I generated a new auth key, this time without ephemeral being enabled, but labeled the containers I have with tsdproxy.ephemeral = true. None of the endpoints came up as ephemeral as I would expect. Am I misunderstanding the implementation?

**Comment by almeidapaulopt** [comment, marked as answer]
They can only be created using ephemeral auth keys (not through the regular authentication flow)...

https://tailscale.com/kb/1111/ephemeral-nodes
